### ISSUE TITLE
Integration with Przelewy24 payment platform and InPost parcel locker service (Poland)

### ISSUE THREAD

**Issue by UnrulyNatives** [binafy/laravel-cart#6]
As an amateur programmer and writer. I currently struggle with creating a solution for selling e-books (delivery by e-mail) and physical books. For the purpose, there is a need to integrate three solutions. Please let me know if you'd be interested to accept my contributions and ideas. Currently there is no Laravel based, ready to use shop. And as this movie suggest, there was c. 5000 persons community interested. This demand would boost your repo's popularity.

Needed features: 
- Integration to Przelewy24, one of Poland' s 2 largest payment processors. 
- Integration with InPost "Paczkomat" service. Delivery to a **parcel locker service** (https://en.wikipedia.org/wiki/Parcel_locker)

Also, I could provide many innovative ideas as an end-user of such web store. 
I can do simple and moderately advanced programming in PHP, but I could also provide a list of ideas collected from publishers, who operate their own solutions for on-line bookstores.

In short, I am offering to assist in creating a working instance of a shop which would be fully viable and ready for any Poland based developer to use.

**Comment by milwad-dev** [comment]
Thanks for your suggestion.
We can't make this repo just for Poland developers, but if you have any ideas for all developers, I welcome them.

**Comment by UnrulyNatives** [comment]
We have a misunderstanding. An on-line cart should be able to accommodate multiple payment processors. 
I could help you to integrate one. I have not said that the Przelewy24 should be the only one. 
Some day someone else would do Stripe or other processors.

**Comment by milwad-dev** [comment]
So we can think about this, but now we need to add some utility.
